src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -212,7 +212,8 @@ private void EnqueueWork(bool highPriority, bool frozenPartialSemantics, Cancell
                 using var linkedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(lastNonFrozenComputationToken, cancellationToken);
                 try
                 {
-                    return await RecomputeTagsAsync(highPriority, frozenPartialSemantics, linkedTokenSource.Token).ConfigureAwait(false);
+                    return await RecomputeTagsAsync(
+                        highPriority, frozenPartialSemantics, blockingJTFCall: false, linkedTokenSource.Token).ConfigureAwait(false);
```

Comment:
we now pass in if we're being blocked in JTF.  Note: i also saw JTF had a IsMainThreadMaybeBlocked helper that we could potentially defer to instead of passing this in.  But i'm not super familiar with JTF to be sure.